boot/freeldr/freeldr/arch/i386/entry.S

```diff
@@ -21,6 +21,9 @@
 #include <arch/pc/x86common.h>
 #include <arch/pc/pcbios.h>
 
+#define PIC1_COMMAND      HEX(20)
+#define PIC_EOI           HEX(20)
```

Comment:
is it correct to use HEX(20) for both defines here?

Reply:
Yeah, there is no difference between 2 of those